desafio_kobe/rick_and_morty_app/pubspec.yaml

```
@@ -0,0 +1,33 @@
+name: rick_and_morty_app
+description: "A new Flutter project."
+
+publish_to: 'none' 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)